WordPress/src/main/res/color/reader_follow_button_text.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
     <item android:color="@color/reader_following" android:state_selected="true" />
-    <item android:color="@color/reader_follow" />
+    <item android:color="@color/reader_follow_blue" />
```

Comment:
It looks like the <b><i>Follow</i></b> button was changed from dark blue to light blue.  The dark blue matched the color of the <b><i>Follow</i></b> option in the menu.  Now they are different.  Show they be the same?  See the screenshots below for example.

![follow](https://cloud.githubusercontent.com/assets/3827611/20375555/10ea2ec8-ac3d-11e6-811e-66cecd5b155d.png)